src/room/GroupCallView.tsx

```diff
@@ -318,7 +318,7 @@ export const GroupCallView: FC<Props> = ({
         client={client}
         matrixInfo={matrixInfo}
         muteStates={muteStates}
-        onEnter={(): void => enterRTCSession(rtcSession, perParticipantE2EE)}
+        onEnter={() => enterRTCSession(rtcSession, perParticipantE2EE)}
```

Comment:
Should the type of `Props.onEnter` in LobbyView.tsx be changed to `() => Promise<void>`?

Or if `onEnter` is meant to ignore the `Promise<void>` return value of `enterRTCSession`, make it explicit with this:
```suggestion
        onEnter={() => void enterRTCSession(rtcSession, perParticipantE2EE)}
```